Chrome extension denyying crx file

Busanga opened the pull request:
Hi, am new to site so not sure if i am in the correct place. 

i put the crx file into chrome extensions but it says this is not allowed and wont allow me to turn it on.
![Screenshot 2021-02-05 195033](https://user-images.githubusercontent.com/78615214/107082025-7bfbb100-67eb-11eb-97f0-92e08ec9429c.jpg)

aciidic committed:
Remove tracking, notifications & permissions

Attempt to remove analytics tracking & externally-delivered notification popups. Also removed some permissions to match older v7.1.6 by deanoemcke (original developer)

Untested and without guarantee!

aciidic committed:
Update README.md

aciidic committed:
Update README.md

aciidic committed:
Update README.md

aciidic committed:
Updated Group Policy documentation

aciidic committed:
Update README.md

aciidic committed:
Group Policy extension installation

Added instructions for installation of local/custom extension .crx file using Microsoft Group Policy

aciidic committed:
Update README.md

aciidic committed:
Update README.md

aciidic committed:
Update README.md

aciidic committed:
Update manifest.json

aciidic committed:
Update manifest.json

aciidic committed:
Update README.md

luke-jr committed:
Support migration from TGS with other extension ids

LewisSpring commented:
You're in the wrong place - you should be in Issues instead. No worries!

I'm, sure you're aware, but The Great Suspender has been removed from the Google Chrome Extension Store since it is potentially malware.
It is warning you because you are sideloading the extension, possibly via a external application by the looks of it. Chrome is trying to protect you from this, as malicious programs often add Extensions as part of their payload.

I'd recommend using The Marvellous Suspender instead, I can't recommend it personally, but the community are moving there.

Busanga commented:
Thank you for the info and heads up. In future will use issues cheers

aciidic committed:
Merge pull request #3 from luke-jr/migrate

Support migration from TGS with other extension ids

aciidic committed:
Update README.md

tbmc committed:
Remove gsAnalytics.js

aciidic committed:
Merge pull request #4 from tbmc/master

Remove gsAnalytics.js which is not referenced in any code anymore

aciidic committed:
Update messages.json

aciidic committed:
Update messages.json

aciidic committed:
Update messages.json

aciidic committed:
Update messages.json

aciidic committed:
Update messages.json

aciidic committed:
Update messages.json

aciidic committed:
Update messages.json

aciidic committed:
Update messages.json

aciidic committed:
Update manifest.json

Update manifest version to 7.1.12

aciidic committed:
Update about.html

aciidic committed:
Option to disable firstrun/update popups

Repurpose "NO_NAG" option. When activated it will disable the popups during first run and extension updates.

aciidic committed:
Option to disable firstrun/update popups

Repurpose "NO_NAG" option. When activated it will disable the popups during first run and extension updates.

aciidic committed:
Option to disable firstrun/update popups

Repurpose "NO_NAG" option. When activated it will disable the popups during first run and extension updates.

tianhuil committed:
Add some annotation to make README a bit more clear

aciidic committed:
Merge pull request #5 from tianhuil/patch-2

Add some annotation to make README a bit more clear

yurovant committed:
Update package-lock.json

aciidic committed:
Merge pull request #6 from yurovant/patch-1

Update package-lock.json
